rest/route/host_create_test.go

```diff
@@ -286,6 +286,69 @@ func TestMakeHost(t *testing.T) {
 	assert.Equal(ec2Settings2.AMI, "ami-987654")
 }
 
+func TestCreateHostRoute(t *testing.T) {
```

Comment:
It's good that we added tests for the route! Can we also test some other cases to make sure it make hosts up to the cap and no more? For example:
* Just running the route with no pre-existing host.create hosts makes the correct number of hosts all at once.
* Running the route with non-zero host.create hosts but fewer than the number requested causes the route to make the remainder when it runs?

Reply:
added